CHANGELOG.md

```diff
@@ -2,5 +2,9 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+
 ## [0.1.0] - 2019-xx-xx
 - Initial release
+- Added methods to project model results onto an Instrument view for both regular
+  and irregularly gridded models (instrument_view_through_model,
+  instrument_view_irregular_model, and instrument_altitude_to_model_pressure)
```

Comment:
Do we need to put things in the changelog before the first release?  I was thinking no.

Reply:
Let's take this out and not update the CHANGELOG until we have an accepted base that we are changing things from.